src/sentry/api/endpoints/team_projects.py

```diff
@@ -214,4 +218,15 @@ def post(self, request: Request, team) -> Response:
                 sender=self,
             )
 
+        # Create project option to turn on ML similarity feature for new EA projects
+        if project:
+            is_seer_eligible_platform = project.platform in SEER_ELIGIBLE_PLATFORMS
+            if (
+                project.organization.flags
+                and project.organization.flags.early_adopter
+                and is_seer_eligible_platform
+                and get_local_region().name == "us"
```

Comment:
cc @getsentry/hybrid-cloud is there a preferred way of doing this?

Reply:
In general, we've done this by using options which can be applied on a per-region basis (and also work nicely for enabling in ST)

Reply:
@jangjodi can we use an option for this? maybe make a boolean that is default false across all regions, then `true` for us?

Reply:
thanks! I'll make this an option instead